playwright/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  playwright-tests:
+    build:
+      context: ./..
+      dockerfile: playwright/Dockerfile
+    volumes:
+      - ../src/__tests__/visual/__screenshoots__:/app/src/__tests__/visual/__screenshoots__
+      - ../playwright-report:/app/playwright-report
+      - ../test-results:/app/test-results
+    command: npx playwright test --reporter=html
+    environment:
+      CI: false
```

Comment:
Just a question, why you're setting this explicitly to false?

Reply:
This is a leftover. I should remove it. Thanks for noticing that 😉